src/App.tsx

```diff
@@ -15,36 +20,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const getPreparedGoods: PreparedGoods = (
+  goods,
+  sortField,
+  isReversedActive,
+) => {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.byName:
+          return good1.localeCompare(good2);
+        case SortType.byLength:
+          return good1.length - good2.length;
+        default:
+          return 0;
```

Comment:
Would be great so move switch...case out of the sort method callback, so you don't need to call it if it's a default case